client/index.html

```diff
@@ -1,11 +1,116 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <template data-type="pug" data-src="./view/base/meta.pug" />
-    <link rel="stylesheet" href="./index.css">
-    <script type="module" src="./index.js" defer></script>
-  </head>
-  <body>
-    <template data-type="pug" data-src="./view/browsers/browsers.pug" />
-  </body>
+<head>
+  <link rel="stylesheet" href="./index.css">
+  <script type="module" src="./index.js" defer></script>
+  <meta name="color-scheme" content="dark light">
+</head>
+
+<body>
+  <article class="Article Article--left">
+    <section class="Introduction">
+      <div class="Introduction__header">
+        <h1 class="Article__h1">
+          <img src="./assets/browserlist_logo.svg" role="presentation">
+          Browserslist
+        </h1>
+        <a class="Badge" href="https://github.com/browserslist/browserslist">
+          <img src="./assets/github_logo.svg" alt="">
+          10.5k
+        </a>
+        <a class="Badge" href="https://twitter.com/Browserslist">
+          <img src="./assets/twitter_logo.svg" alt="">
+          4k
+        </a>
+      </div>
+      <p class="Introduction__about">Shared browser and platform compatibility config for popular JavaScript tools,
+        including
+        Autoprefixer, Babel, ESLint, PostCSS, and Webpack</p>
+      <p class="Introduction__supported">
+        Supported by
+        <img src="./assets/evil_maritians_logo.svg" role="presentation">
+        <a href="" class="Link">Evil Martians</a> and
+        <img src="./assets/cube_logo.svg" role="presentation">
+        <a href="" class="Link">Cube</a>
+      </p>
+    </section>
+    <h2>How to get started</h2>
+    <p>
+      <a data-query="defaults"><code>defaults</code></a> - if you're building a web application for the global audience.
+    </p>
+    <p>
+      <a data-query="node 18"><code>node 18</code></a> - if you're building a Node.js application, e.g., for SSR.
+    </p>
+    <p><a href="" class="Link">Autoprefixer</a> Autoprefixer ,
+      <a href="" class="Link">Babel</a> or any of the following libraries will find target browsers automatically, when you add the
+      following to
+      <b>package.json</b>:
+    </p>
+    <a data-query="defaults, not IE 11, maintained node versions"><pre class="Pre">
+      <code>"browserslist": [</code>
+      <code>  "defaults",</code>
+      <code>  "not IE 11",</code>
+      <code>  "maintained node versions"</code>
+      <code>]</code>
+    </pre></a>
+    <p>You can also check publicly Browserslist configs on <a href="" class="Link">GitHub</a></p>
+
+    <h2>Query Syntax</h2>
+    <p> You can specify the browser and Node.js versions by queries (case insensitive):</p>
+    <ul></ul>
```

Comment:
Misspell. `<ul>` closed where it was not supposed to

Reply:
It is the part, which should be replaced by real content))